infra/src/main/kotlin/fi/fta/geoviite/infra/configuration/WebConfiguration.kt

```diff
@@ -56,7 +58,10 @@ import org.springframework.web.servlet.config.annotation.WebMvcConfigurer
 @ConditionalOnWebApplication
 @EnableWebMvc
 @Configuration
-class WebConfig : WebMvcConfigurer {
+class WebConfig(
+    private val frameConverterRequestConverterV1: FrameConverterRequestConverterV1,
+    private val frameConverterListRequestConverterV1: FrameConverterListRequestConverterV1
+) : WebMvcConfigurer {
```

Comment:
Näitä ei ollut aiemmin, jos tuohon pyynnön deserialisointiin on parempia idiksiä niin mietiskellään

Reply:
Ei ehkä tule parempia ajatuksia, mutta voisi käydä vielä läpi dailyssä keskustellen niin ymmärtää ongelman vähän paremmin.